packages/atlas/lib/src/atlas.dart

```diff
@@ -33,6 +33,11 @@ class Atlas extends StatelessWidget {
   /// which can be used to manipulate the map.
   final ArgumentCallback<AtlasController> onMapCreated;
 
+  /// When the camera position is changed
+  /// Callback Receiving Camera Location
+  /// Recall during map manipulation
```

Comment:
Thoughts on using onCameraPositionChanged for consistency?

Reply:
ok i will fix that